nano_graphrag/_llm.py

```diff
@@ -57,6 +58,7 @@ async def gpt_4o_mini_complete(
 
 
 @wrap_embedding_func_with_attrs(embedding_dim=1536, max_token_size=8192)
+@retry(stop=stop_after_attempt(5), wait=wait_exponential(multiplier=1, min=4, max=10))
```

Comment:
How `retry` detects the error? It seems like any type of Error will cause a retry.
I think probably introducing `retry` here and so `tenacity` in deps is not necessary.